charts/wardend/values.yaml

```
@@ -43,6 +43,7 @@ cosmovisor:
   unsafeSkipBackup: true
 
 node:
+  home: /data/.warden
   enabled: true
   ready: true
   name: node
```

I'd move this to more logical place.

```suggestion
  enabled: true
  ready: true
  name: node
  home: /data/.warden
```